libtransmission/torrent.cc

```diff
@@ -2213,7 +2213,10 @@ void tr_torrent::onTrackerResponse(tr_tracker_event const* event)
         break;
 
     case tr_tracker_event::Type::Warning:
-        tr_logAddWarnTor(this, fmt::format(_("Tracker warning: '{warning}'"), fmt::arg("warning", event->text)));
+        tr_logAddWarnTor(
+            this,
+            fmt::format(_("Tracker warning: '{warning}'"), fmt::arg("warning", event->text))
+                .append(fmt::format(" ({url})", fmt::arg("url", tr_urlTrackerLogName(event->announce_url)))));
```

Comment:
_(nit)_ no need to use arg syntax for non-i18nized calls

```suggestion
                .append(fmt::format(" ({})", tr_urlTrackerLogName(event->announce_url))));
```